Subject: [PATCH] QS: Bluetooth easy toggle [2/2]

Add an option to enable single click on the Bluetooth tile to turn it on or off

diff --git a/res/values/cr_strings.xml b/res/values/cr_strings.xml
index 812d4f1..d74b623 100644
--- a/res/values/cr_strings.xml
+++ b/res/values/cr_strings.xml
@@ -31,4 +31,8 @@
     <string name="qs_advanced_data_title">Advanced data tile</string>
     <string name="qs_advanced_data_summary">Enable one tap action to enable or disable data</string>
 
+    <!-- Advanced bt tile -->
+    <string name="qs_advanced_bt_title">Advanced bluetooth tile</string>
+    <string name="qs_advanced_bt_summary">Enable one tap action to enable or disable bluetooth</string>
+
 </resources>
diff --git a/res/xml/status_bar_settings.xml b/res/xml/status_bar_settings.xml
index eb0b7d5..1f67837 100644
--- a/res/xml/status_bar_settings.xml
+++ b/res/xml/status_bar_settings.xml
@@ -144,6 +144,12 @@
             android:summary="@string/qs_advanced_data_summary"
             android:defaultValue="false" />
 
+        <cyanogenmod.preference.SecureSettingSwitchPreference
+            android:key="qs_bt_easy_toggle"
+            android:title="@string/qs_advanced_bt_title"
+            android:summary="@string/qs_advanced_bt_summary"
+            android:defaultValue="false" />
+
     </PreferenceCategory>
 
 </PreferenceScreen>
